documentation/library_guide/common_cross_document_links.txt

```diff
@@ -0,0 +1,36 @@
+CMake Support

```

Comment:
"CMake Support" is rendered as a text on each HTML page, and so do other "headings" in this file. Either need to convert them to comments or simply remove, as these are not of much value, especially if the "macro" names are chosen to be self-descriptive.

Reply:
Let's remove them. I had added them to keep track of where I was in the files, but I don't think that we need to do that moving forward.